### DISCUSSION TITLE
Cannot build because of YubiKey

### DISCUSSION THREAD

**Discussion by DreamTexX** [keepassxreboot/keepassxc#6735] [Q&A] [answered]
Hey, don't wanna open an issue as it may be a simple problem...
i try to build the source from the latest release and even with yubikey libs installed via apt and with `-DWITH_XC_YUBIKEY=OFF` i get the following error:

```
-- Found Git HEAD Revision: 9c108b9

-- Setting up build for KeePassXC v2.6.6

CMake Error at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find YubiKey (missing: YUBIKEY_LIBRARIES YUBIKEY_INCLUDE_DIRS)
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindYubiKey.cmake:25 (find_package_handle_standard_args)
  CMakeLists.txt:462 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/kevin/downloads/keepassxc-2.6.6/build/CMakeFiles/CMakeOutput.log".
See also "/home/kevin/downloads/keepassxc-2.6.6/build/CMakeFiles/CMakeError.log".
```

Anyone with the same problem? How can i solve this

**Comment by droidmonkey** [comment, marked as answer]
Please follow the setup instructions: https://github.com/keepassxreboot/keepassxc/wiki/Set-up-Build-Environment-on-Linux#install-the-optional-dependencies

**Reply by DreamTexX** [reply]
But why is it complaining about it even if i disable yubikey?

**Reply by droidmonkey** [reply]
If you have -DWITH_XC_ALL=ON then it will override the individual setting. Set all to off and turn on the ones you want.

**Reply by DreamTexX** [reply]
Like ive written, i disabled yubikey with the Setting but it still complains

**Reply by DreamTexX** [reply]
That is the thing i dont understand. Why is it complaining when i disabled it?

**Reply by droidmonkey** [reply]
You have not disabled it. You will need to issue a new cmake command like: `cmake -DWITH_XC_ALL=OFF -DWITH_XC_YUBIKEY=OFF ..`
